docs/favicons/manifest.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "docs.nunit.org",
+    "short_name": "nunit docs",
```

```suggestion
    "short_name": "NUnit Docs",
```

⛏️ Not sure if this has to be lower-case. If we have flexibility on casing, NUnit Docs would be better.

---

Done